setup/configure_tpm_emulator/test.sh

```diff
@@ -12,27 +12,45 @@ if [ "$RUNNING" != "0" -a "$RUNNING" != "1" ]; then
 fi
 
 TPM_EMULATOR=ibm-tpm-emulator
-rlIsFedora && TPM_EMULATOR=swtpm
+SELINUX_POLICY=ibmswtpm_permissive
+rlIsFedora && TPM_EMULATOR=swtpm && SELINUX_POLICY=swtpm_permissive
+
+setup_selinux_policy() {
+    rlRun "yum -y install policycoreutils selinux-policy-devel"
+    if [ -e "${SELINUX_POLICY}".te ] && ! semodule -l | grep -q "${SELINUX_POLICY}"; then
+        rlRun "make -f /usr/share/selinux/devel/Makefile ${SELINUX_POLICY}.pp"
+        rlAssertExists "${SELINUX_POLICY}.pp"
+        rlRun "semodule -i ${SELINUX_POLICY}.pp"
+    fi
+    # allow tpm2-abrmd to connect to swtpm port
+    rlRun "setsebool -P tabrmd_connect_all_unreserved on"
+}
 
 rlJournalStart
 
     rlPhaseStartSetup "Install TPM emulator"
         rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
         # for RHEL or CentOS Stream we use Sergio's build of IBM TPM emulator
+        baseurl='https://download.copr.fedorainfracloud.org/results/scorreia/keylime/fedora-$releasever-$basearch/'
```

Comment:
We probably don't need it; if I recall correctly, cfssl is not in the Fedora repos, but then I don't think we use it anyway in the tests (or do we?)

Reply:
I am not sure.. Wasn't it necessary for ibmswtpm2 to build? Otherwise I agree we should not need it.

Reply:
Nah, not for the build. It's another ssl provider, as an alternative to openssl. Keylime supports both, so I added it there in case we want to support/test it as well.